hack/operatorhub/config.yaml

```diff
@@ -1,5 +1,5 @@
-newVersion: 2.12.0
-prevVersion: 2.11.1
+newVersion: 2.12.1
+prevVersion: 2.12.0
```

Comment:
Sorry for that very late comment, but did we actually publish `2.12.0` on the operator hub?

<img width="1453" alt="image" src="https://github.com/elastic/cloud-on-k8s/assets/976373/894ed3b5-f4d5-4417-abc8-907caf8f945b">